pipelines/LSSTComCamSim/DRP-ops-rehearsal-3.yaml

```diff
@@ -0,0 +1,241 @@
+description: DRP Pipeline for Simulated ComCam Ops Rehearsal 3
+instrument: lsst.obs.lsst.LsstComCamSim
+imports:
+  - $DRP_PIPE_DIR/pipelines/_ingredients/DRP-full.yaml
+  - $ANALYSIS_TOOLS_DIR/pipelines/coaddQualityCore.yaml
+  - $ANALYSIS_TOOLS_DIR/pipelines/matchedVisitQualityCore.yaml
+  - $ANALYSIS_TOOLS_DIR/pipelines/coaddQualityExtended.yaml
+  - $ANALYSIS_TOOLS_DIR/pipelines/visitQualityCore.yaml
+parameters:
+  filterMap: {
+    "g": "lsst_g",
+    "r": "lsst_r",
+    "i": "lsst_i",
+  }
+tasks:
+  selectGoodSeeingVisits:
+    class: lsst.pipe.tasks.selectImages.BestSeeingQuantileSelectVisitsTask
+    config:
+      connections.goodVisits: goodSeeingVisits
+      nVisitsMin: 12
+      # TODO: Check nImages that 4 days is sufficient at pixel level
+      minMJD: 60390.00
+      maxMJD: 60394.00
+  updateVisitSummary:
+    class: lsst.drp.tasks.update_visit_summary.UpdateVisitSummaryTask
+    config:
+      # TODO: Evaluate GBDES and skyCorr after initial DRP
+      wcs_provider: "input_summary"
```

Comment:
We may want to note the TODO here for fgcmcal after first round, and in the following line after skycorr evaluation.